keystore-manager.sh

```
@@ -282,6 +354,771 @@ encode_base64() {
+update_ios_section() {
+    local SECRETS_FILE="$ENV_FILE"
+
+    for key in "${!IOS_STRING_SECRETS[@]}"; do
+        local value="${IOS_STRING_SECRETS[$key]}"
+
+        # Check if key exists in file
+        if grep -q "^${key}=" "$SECRETS_FILE"; then
+            # Update existing key
+            if [[ -n "$value" ]]; then
+                # Replace with new value (escape special characters)
+                local escaped_value=$(printf '%s\n' "$value" | sed 's/[&/\]/\\&/g')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)